### ISSUE TITLE
Text Alternative for icon is not read

### ISSUE THREAD

**Issue by SanketMohite07** [UI5/webcomponents-react#8256]
### Is your feature request related to a problem?

Text alternatives are not being read out by the screen reader Although text alternatives are present for the icons in the navigation bar.

<img width="455" height="214" alt="Image" src="https://github.com/user-attachments/assets/d5d8f06d-d34d-43c3-927e-661445bc6a87" />

SandBox - https://codesandbox.io/p/sandbox/falling-river-yrvzd9

### Describe the solution you'd like

Each focusable item should have a programmatic name announced by the screen reader on focus and selection, even when rendered without visible text

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @SanketMohite07
as mentioned in my previous message on the other issue you raised, v1 reached end-of-support in July 2025, so we're generally unable to provide support for outdated versions.

That said, I did take a quick look at the behavior in both v2 and the latest v1 release, and I wasn't able to reproduce the reported issue in either. When raising issues, please ensure you're testing against an up-to-date version and that the problem hasn't already been resolved.
